spec/models/blog_post_spec.rb

```diff
@@ -59,6 +59,24 @@
     end
   end
 
+  describe "unscoped_orderly" do
+    it "returns unpublished blog posts" do
+      unpublished_post = create(:blog_post, :published_at => nil)
+      expect(BlogPost.unscoped_orderly).to eq([unpublished_post])
+    end
+
+    it "returns published blog posts" do
+      published_post = create(:blog_post, :published_at => Time.now)
+      expect(BlogPost.unscoped_orderly).to eq([published_post])
+    end
+
+    it "returns the last created blog post first" do
+      first_post = create(:blog_post)
+      second_post = create(:blog_post)
+      expect(BlogPost.unscoped_orderly).to eq([second_post, first_post])
```

Comment:
I'm not really sure it's worth re-testing Rails functionality.

I think a more useful test would be to test the message sending:
```
unscoped = double
allow(BlogPost).to receive(:unscoped).and_return(unscoped)
expect(unscoped).to receive(:order).with("created_at DESC")
BlogPost.unscoped_desc
```